lib/broadside/configuration.rb

```diff
@@ -36,9 +38,36 @@ def ecs
       @ecs ||= EcsConfig.new
     end
 
+    def ssh_cmd(ip, options = { tty: false })
+      cmd = 'ssh -o StrictHostKeyChecking=no'
+      cmd << ' -t -t' if options[:tty]
+      cmd << " -i #{@ssh[:keyfile]}" if @ssh[:keyfile]
+      if @ssh[:proxy]
+        cmd << " -o ProxyCommand=\"ssh #{@ssh[:proxy][:host]} nc #{ip} #{@ssh[:proxy][:port]}\""
+      end
+      cmd << " #{@ssh[:user]}@#{ip}"
```

Comment:
or maybe better yet, add a custom 
```ruby
validates_each :ssh do |record, attr, val|
  ...
end
```
i got rid of verify on my branch though so if you just want to wait i'll add the validation